packages/discord.js/src/structures/Webhook.js

```diff
@@ -393,6 +403,22 @@ class Webhook {
     return this.avatar && this.client.rest.cdn.Avatar(this.id, this.avatar, options);
   }
 
+  /**
+   * Whether this webhook is created by a user.
+   * @returns {boolean}
+   */
+  isUserCreated() {
+    return Boolean(this.type === WebhookType.Incoming && this.owner && !this.owner.bot);
+  }
+
+  /**
+   * Whether this webhook is created by an application.
+   * @returns {boolean}
+   */
+  isApplicationCreated() {
+    return this.type === WebhookType.Application;
+  }
```

Comment:
The type it's checking doesn't match with the types.

Reply:
Why not?

Reply:
You fixed this in https://github.com/discordjs/discord.js/pull/7317/commits/79fed2fe1304175c2ac922dd50c7f44a24588429.

Reply:
Oh okay, forgot about making that commit.